### PR TITLE
Add test for multiple model versions

### PR DESCRIPTION
**Pull request by szalpal**
Signed-off-by: szalpal <mszolucha@nvidia.com>

### CODE REVIEW

**Review by awolant — DISMISSED** [review]

File: qa/L0_many_versions/many_versions_client.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+    models = {
+        "dali_all": [1, 2, 3],
+        "dali_latest": [2, 3],
+        "dali_specific": [2, 3],
+    }
+
+    for name, versions in models.items():
+        for ver in versions:
+            if not triton_client.is_model_ready(name, str(ver)):
```
Comment (banasraf) [inline qa/L0_many_versions/many_versions_client.py:63]: Maybe test also if the versions that shouldn't be loaded aren't loaded?

File: qa/L0_many_versions/many_versions_client.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    models_not_loaded = {
+        "dali_latest": [1],
+        "dali_specific": [1],
+    }
+
+    for name, versions in models_loaded.items():
+        for ver in versions:
+            if not triton_client.is_model_ready(name, str(ver)):
+                print("FAILED: Model {} version {} not ready".format(name, ver))
+                sys.exit(1)
```
Comment (banasraf) [inline qa/L0_many_versions/many_versions_client.py:70]:
```suggestion
    versions = [1, 2, 3]
    for name, exp_vers in models_loaded.items():
        for ver in versions:
            if not triton_client.is_model_ready(name, str(ver)) and ver in exp_vers:
                print("FAILED: Model {} version {} not ready".format(name, ver))
                sys.exit(1)
            if triton_client.is_model_ready(name, str(ver)) and ver not in exp_vers:
                print("FAILED: Model {} version {} loaded unexpectedly".format(name, ver))
                sys.exit(1)
```

One loop and no need for `models_not_loaded` ?

Comment (szalpal) [reply qa/L0_many_versions/many_versions_client.py:70]: I'd rather have the 2 loops: makes code more readable and that's good in test ;)

File: qa/L0_many_versions/many_versions_client.py (hunk 1 above)
Comment (szalpal) [reply qa/L0_many_versions/many_versions_client.py:68]: Done

**Review by banasraf — APPROVED** [review]

**Review by awolant — APPROVED** [review]

**Review by banasraf — APPROVED** [review]

**Pull request merged**